### ISSUE TITLE
Rounding issue with limiting decimals for numbers near 100 million

### ISSUE THREAD

**Issue by james-rabbit** [JsCommunity/human-format#18]
I found some cases where `human-format` breaks, it seems to have an internal rounding error somewhere and it outputs the wrong number of digits:

> ```javascript
> humanFormat(111111111, { decimals: 1 })
> ```
> returns: `111.09999999999998 M`
> expected: `111.1 M`

> ```javascript
> humanFormat(100000001, { decimals: 1 })
> ```
> returns: `99.99999999999999 M`
> expected: `100.0 M`

**Comment by james-rabbit** [comment]
I was able to work around this issue by using `humanFormat.raw`:
```javascript
format(number, decimals = 1) {
    const rawFormat = humanFormat.raw(number);
    return `${rawFormat.value.toFixed(decimals)} ${rawFormat.prefix}`;
}
```

**Comment by julien-f** [comment]
Thanks for your report 😃 

I have opened #19 with a test for this, I'll take a look when I can, feel free to take of it if you want.

**Comment by julien-f** [comment]
The fix has been released in 0.9.1, thank you!
